src/Chae/Tree.elm

```diff
@@ -182,7 +182,7 @@ First argument is `Maybe Id` is ether:
 - `Nothing` => result is given tree (with empty ancestors `List`).
 - `Just parentId` => result is sub tree for node with `id == parentId`.
 
-Returns tuple containing sub tree and list of ancestors of `Node`.
+Returns tuple containing sub tree and list of ancestors (paratenrs of root `Node`).
```

Comment:
i like how you describe the order of the list, but since the first element of the tuple is a sub tree (`Tree a`), mentioning `Node` doesn't sit exactly right. How about `...and list of ancestors of that sub tree (from parent to root)`?